From c6c6c2d5c70b94b6cbf845d975c2f66759d0e853 Mon Sep 17 00:00:00 2001
From: Stanislaw Gruszka <sgruszka@redhat.com>
Date: Fri, 5 Sep 2014 11:23:41 -0400
Subject: [kernel] if_ether: add IEEE 802.21 Ethertype

Message-id: <1409916232-5882-5-git-send-email-sgruszka@redhat.com>
Patchwork-id: 91434
O-Subject: [RHEL7 PATCH 04/15] if_ether.h: add IEEE 802.21 Ethertype
Bugzilla: 1136324
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: John Greene <jogreene@redhat.com>

BZ#1136324

Backport of:

commit b62faf3cdc875a1ac5a10696cf6ea0b12bab1596
Author: Simon Wunderlich <sw@simonwunderlich.de>
Date:   Mon Mar 3 17:23:10 2014 +0100

    if_ether.h: add IEEE 802.21 Ethertype

    Add the Ethertype for IEEE Std 802.21 - Media Independent Handover
    Protocol. This Ethertype is used for network control messages.

    Signed-off-by: Simon Wunderlich <sw@simonwunderlich.de>
    Signed-off-by: Mathias Kretschmer <mathias.kretschmer@fokus.fraunhofer.de>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Brew:
https://brewweb.devel.redhat.com/taskinfo?taskID=7918052

Testing:
Tested on Intel, Atheros, Ralink wireless hardware.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/linux/if_ether.h b/include/uapi/linux/if_ether.h
index ade07f1..d90bbe5 100644
--- a/include/uapi/linux/if_ether.h
+++ b/include/uapi/linux/if_ether.h
@@ -88,6 +88,7 @@
 #define ETH_P_FCOE 0x8906  /* Fibre Channel over Ethernet  */
 #define ETH_P_TDLS 0x890D          /* TDLS */
 #define ETH_P_FIP 0x8914  /* FCoE Initialization Protocol */
+#define ETH_P_80221 0x8917  /* IEEE 802.21 Media Independent Handover Protocol */
 #define ETH_P_QINQ1 0x9100  /* deprecated QinQ VLAN [ NOT AN OFFICIALLY REGISTERED ID ] */
 #define ETH_P_QINQ2 0x9200  /* deprecated QinQ VLAN [ NOT AN OFFICIALLY REGISTERED ID ] */
 #define ETH_P_QINQ3 0x9300  /* deprecated QinQ VLAN [ NOT AN OFFICIALLY REGISTERED ID ] */
-- 
1.7.1